examples/job_attachments/snapshot_glob.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+####
+# This script demonstrates building on Job Attachments CLI primitives for manifest and attachment handling.
+# In this example, we snapshot a directory, make a modification and diff against the snapshot to show to find what files have changed.
+# This example is usefule as a cron job to create incrementally updated manifests and upload only modified or new files.
+###
```

Comment:
This looks like the same documentation as above. We should update it to include some discussion on the globing feature.

Reply:
Yup, updated.